pom.xml

```diff
@@ -65,7 +65,7 @@
         <gravitee-exchange.version>1.8.2</gravitee-exchange.version>
         <gravitee-expression-language.version>3.2.1</gravitee-expression-language.version>
         <gravitee-fetcher-api.version>2.0.0</gravitee-fetcher-api.version>
-        <gravitee-gateway-api.version>3.5.0</gravitee-gateway-api.version>
+        <gravitee-gateway-api.version>3.5.1-apim-6600-add-envid-in-subscription-350-SNAPSHOT</gravitee-gateway-api.version>
```

Comment:
Don't forget to replace with the released version.

Reply:
Done